packages/lib/src/vis/heatmap/HeatmapVis.module.css

```diff
@@ -1,4 +1,5 @@
 .root {
+  grid-area: vis;
```

Comment:
Could this get in the way of uses of `HeatmapVis`/`LineVis`... by themselves in consumer apps ?